### PR TITLE
Fixed tutorial documentation

### PR DESCRIPTION
**Pull request by alithethird**
Applicable spec: <link>

### Overview

Updated documentation to add PostgreSQL extensions and also updated the integration commands.

### Rationale

The tutorial was not able to create a working charm so had to be updated.

### Checklist

- [x] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
- [x] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
- [x] The changes are compliant with [ISD054 - Managing Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
- [x] The documentation is generated using `src-docs`
- [x] The documentation for charmhub is updated
- [x] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)

### CODE REVIEW

File: docs/tutorial.md [hunk 1]
```diff
@@ -64,10 +64,17 @@ indico/0*                 waiting   idle   10.1.74.70             Waiting for da
 Provide integration between Indico and PostgreSQL:
 
 ```bash
-juju integrate indico postgresql-k8s:db
+juju integrate indico postgresql-k8s:database
 ```
 
-Note: `db` is the name of the relation. This is needed because establishes that the two charms are compatible with each other.  You can run `juju info indico` to check what are the relation names required by the Indico application.
+Note: `database` is the name of the relation. This is needed because establishes that the two charms are compatible with each other.  You can run `juju info indico` to check what are the relation names required by the Indico application.
```
Comment (cbartz) [inline docs/tutorial.md:70]: I would use the term "integration" here, because `relation` is the legacy.

**Comment by cbartz** [comment]
Can you also complete the checklist and add the appropriate labels (trivial and documentation)? Thank you very much.

File: docs/tutorial.md (hunk 1 above)
Comment (alithethird) [reply docs/tutorial.md:70]: Thank you!

**Review by mthaddon — APPROVED** [review]

**Review by cbartz — APPROVED** [review]
LGTM, thanks!

**Pull request merged**